pkg/kuberang/mainworkflow.go

```
@@ -174,10 +178,14 @@ func CheckKubernetes(skipCleanup bool) error {
 	}
 
 	// 4. Check internet connectivity from pod
-	if ko := RunKubectl("exec", busyboxPodName, "--", "wget", "-qO-", "Google.com"); busyboxPodName == "" || ko.Success {
-		util.PrettyPrintOk(out, "Accessed Google.com from BusyBox")
+	if !config.SkipDNSTests {
+		if ko := RunKubectl("exec", busyboxPodName, "--", "wget", "-qO-", "Google.com"); busyboxPodName == "" || ko.Success {
```

The error is ignored.
```
Accessed Google.com from BusyBox                                                [ERROR IGNORED]
```